.github/workflows/daily-boot-iso.yml

```diff
@@ -0,0 +1,42 @@
+name: Build daily Rawhide+COPR boot.iso
+on:
+  schedule:
+    - cron: 0 22 * * *
+  # be able to start this action manually from a actions tab when needed
+  workflow_dispatch:
+
+jobs:
+  boot_iso:
+    name: Build boot.iso
+    runs-on: ubuntu-latest
+    steps:
+      # lorax and its dependencies do not exist in Ubuntu, and we want to test the latest Fedora version anyway
+      # lorax does mounts and uses loop devices, thus needs to be privileged
+      - name: Start fedora container
+        run: |
+          mkdir data
+          docker run --name fedora -d --privileged --network host -v $PWD/data:/data fedora:latest sleep infinity
+
+      - name: Install lorax
+        run: docker exec fedora dnf install -y lorax
+
+      - name: Run lorax
+        run: docker exec -w /data fedora lorax -p Fedora -v 34 -r 34 -s http://download.fedoraproject.org/pub/fedora/linux/development/rawhide/Everything/x86_64/os/ -s https://download.copr.fedorainfracloud.org/results/rpmsoftwaremanagement/dnf-nightly/fedora-rawhide-x86_64/ -s https://download.copr.fedorainfracloud.org/results/@rhinstaller/Anaconda/fedora-rawhide-x86_64/ -s https://copr-be.cloud.fedoraproject.org/results/@storage/blivet-daily/fedora-rawhide-x86_64/ /data/results
```

Comment:
I wonder that it's that simple to call Lorax from container because in general that is pain. However, if this works than it's great!

Reply:
It really is, as lorax is using loop devices and mounts. Hence the need for `--privileged` (see the comment in `steps:`).

Reply:
ohh, I missed the privileged. In that case, yes, it will work :)